vars/common-vars.yml

```diff
@@ -47,3 +48,11 @@ suse_airship_components_image_tag: "{%- if suse_airship_build_local_images or su
                                     {%- else -%}
                                       latest
                                     {%- endif -%}"
+suse_osh_image_version: "pike"
```

Comment:
you are duplicating this value. Shouldn't we all use the same value for osh+airship? If not, cant we name them differently? because on the osh-deploy role in the vars there is the same var which one will be used when we deploy X or Y? This is confusing and could lead to problems so please use a different var name or lets agree in using the same version for both airship and osh

Reply:
Airship+OSH deploy is using Pike on Ubuntu for development an susecon demo. I think OSH only deployment is working on Rocky on Leap.  I see the default value in deploy-osh is set to Ocata. Do you only use Ocata on Ubuntu? I mentioned in the comments to Jiri, Pike on Ubuntu has worked without issue for us. Can we move to Pike on Ubuntu on the osh deployment as well?  Do you want me to change that?

If we can't or have reasons not to align, I am not against to use different variable names either.